tester_app/client/app/bundles/HelloWorld/reducers/index.jsx

```diff
@@ -0,0 +1,10 @@
+import helloWorldReducer from './helloWorldReducer';
+import { $$initialState as $$helloWorldState } from './helloWorldReducer';
```

Comment:
Maybe we can use the docs directory to have a folder like this:

/docs/gen-notes

and we put such a note here:
/docs/gen-notes/reducers

Also, please change words [properties, guestLists, memberships] to [users, blogPosts, comments] to be more generic, @robwise.